### PR TITLE
Print compressoptions

### PR DESCRIPTION
**Pull request by cgzones**

### CODE REVIEW

File: logrotate.c [hunk 1]
```diff
@@ -752,7 +752,7 @@ static int compressLogFile(const char *name, const struct logInfo *log, const st
     char *prevCtx;
     pid_t pid;
 
-    message(MESS_DEBUG, "compressing log with: %s\n", log->compress_prog);
+    message(MESS_DEBUG, "compressing log with %s: %s\n", compress_options_count ? "added options" : "default", log->compress_prog);
```
Comment (kdudka) [inline logrotate.c]: This should have been `log->compress_options_count` I guess.

File: logrotate.c [hunk 2]
```diff
@@ -767,7 +767,7 @@ static int compressLogFile(const char *name, const struct logInfo *log, const st
     sprintf(compressedName, "%s%s", name, log->compress_ext);
 
     if ((inFile = open(name, O_RDWR | O_NOFOLLOW)) < 0) {
-        message(MESS_ERROR, "unable to open %s for compression\n", name);
+        message(MESS_ERROR, "unable to open %s for compression: %s\n", name, strerror(errno));
```
Comment (kdudka) [inline logrotate.c]: This change makes sense.

File: logrotate.c [hunk 3]
```diff
@@ -752,7 +752,7 @@ static int compressLogFile(const char *name, const struct logInfo *log, const st
     char *prevCtx;
     pid_t pid;
 
-    message(MESS_DEBUG, "compressing log with: %s\n", log->compress_prog);
+    message(MESS_DEBUG, "compressing log with %s: %s\n", log->compress_options_count ? "added options" : "default", log->compress_prog);
```
Comment (kdudka) [inline logrotate.c]: The updated message sounds confusing to me:

> compressing log with default: /bin/gzip

I would rather understand it as if /bin/gzip was the default compress program.

What is the motivation behind this change?

If we want to increase the verbosity, we should rather print the array passed to `execvp()`.  But is it really needed? 

The original string that the array is build from is already printed while parsing the configuration...

Comment (cgzones) [reply logrotate.c]: Yeah, you're right; this is not needed.

**Comment by kdudka** [comment]
Thanks for the quick update!  Merging...
